src/test/java/org/folio/rest/impl/AccountsActionChecksAPITests.java

```diff
@@ -30,6 +31,14 @@ public void checkWaiveAmountShouldBeAllowed() {
     actionCheckAmountShouldBeAllowed(accountsCheckWaiveClient, accountToPost);
   }
 
+  @Test
+  public void checkTransferAmountShouldBeAllowed() {
+    Account accountToPost = postAccount();
```

Comment:
What else do we need from AccountsAPITest?

Reply:
updated